Xamarin.Forms.Platform.WinRT/NavigationPageRenderer.cs

```diff
@@ -189,7 +177,11 @@ public void SetElement(VisualElement element)
 				LookupRelevantParents();
 				UpdateTitleColor();
 				UpdateNavigationBarBackground();
-                UpdateToolbarPlacement();
+
+#if WINDOWS_UWP
```

Comment:
this one isn't required. you could have UpdateToolBarPlacement defined to no-op on the platforms not supporting it